docs/serving/cluster-local-route.md

```diff
@@ -29,25 +29,25 @@ inside the cluster:
 
 To configure a KService to only be available on the cluster-local network (and
 not on the public Internet), you can apply the
-`serving.knative.dev/visibility=cluster-local` label to the KService, Route or 
+`networking.knative.dev/visibility=cluster-local` label to the KService, Route or
```

Comment:
I think KService --> Knative service as well here for consistency, and route, service etc probably don't need to have caps throughout

Reply:
Also this was here long before :)